### ISSUE TITLE
SDK 35: Program type already present: com.facebook.react.uimanager.UIManagerReanimatedHelper

### ISSUE THREAD

**Issue by Luckygirlllll** [expo/expo#5893]
### Environment

```
 Expo CLI 3.1.2 environment info:
    System:
      OS: macOS 10.14.6
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 10.15.0 - /usr/local/bin/node
      Yarn: 1.12.3 - /usr/local/bin/yarn
      npm: 6.4.1 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.5 AI-191.8026.42.35.5791312
      Xcode: 11.0/11A420a - /usr/bin/xcodebuild
    npmPackages:
      expo: ^35.0.0 => 35.0.0 
      react: 16.8.3 => 16.8.3 
      react-native: https://github.com/expo/react-native/archive/sdk-35.0.0.tar.gz => 0.59.8 
      react-navigation: ^3.2.1 => 3.13.0 
    npmGlobalPackages:
      expo-cli: 3.1.2
```

### Steps to Reproduce

create Android apk using new expo and expoKit SDK 35 

### Expected Behavior

I can create apk

### Actual Behavior

I'm getting an apk/bundle build error:

>  'Program type already present: com.facebook.react.uimanager.UIManagerReanimatedHelper'

**Comment by lc3t35** [comment]
As per #5820 remove the jar from package where the error raise.

**Comment by Luckygirlllll** [comment]
@lc3t35 and where is this` jar` exactly?

**Comment by BJW** [comment]
@Luckygirlllll I had the same question. For me the location was in the error itself. In the node_modules -> react-native-reanimated -> android -> src -> main -> java -> com -> facebook -> react -> uimanager

**Comment by lc3t35** [comment]
@Luckygirlllll I removed it using Android Studio, in the list of library, you have react-native-reanimated, then if you unfold it, you can see the uimanager, select it and right click Delete.

**Comment by Luckygirlllll** [comment]
I don't have any jar file there, only `UIManagerReanimatedHelper.java`

<img width="451" alt="Screen Shot 2019-10-08 at 12 23 14" src="https://user-images.githubusercontent.com/10096433/66388437-6b081900-e9c6-11e9-8c4b-cabf9c92c17b.png">

**Comment by BJW** [comment]
@Luckygirlllll I think that was the one. You can just temporarily move it outside of your main dir to test the APK. No harm done if it wasn't correct. Move it back. I'm not sure what the GUI is actually doing when you do it from Android Studio but @lc3t35 recommendation might do some extra steps with references that I missed by just removing the file. You could also duplicate the entire folder, try both ways, and DIFF it to compare if it was just removing that file or changing some config settings too. For me just deleting that file allowed it to compile correctly.

**Comment by Luckygirlllll** [comment]
After deleting that file, I'm getting build error: `"error: cannot find symbol variable UIManagerReanimatedHelper`
"

**Comment by pencilcheck** [comment]
Also occuring in SDK 34, is there any legitimate fix for this? instead of this hackish way of removing a file manually from node_modules?

**Comment by RolfNiemand** [comment]
> Solved it here: #5820
> 
> Also occuring in SDK 34, is there any legitimate fix for this? instead of this hackish way of removing a file manually from node_modules?

Thanks - your solution worked - delete the entire folder
